hazelcast/src/hazelcast/client/config.cpp

```diff
@@ -278,11 +311,22 @@ namespace hazelcast {
                 return *this;
             }
 
+            connection_retry_config &client_connection_strategy_config::get_retry_config() {
+                return retry_config_;
+            }
+
+            client_connection_strategy_config &
+            client_connection_strategy_config::set_retry_config(const connection_retry_config &retry_config) {
+                retry_config_ = retry_config;
```

Comment:
You can take by value, and then move:
```suggestion
            client_connection_strategy_config::set_retry_config(connection_retry_config retry_config) {
                retry_config_ = std::move(retry_config);
```